fiftyone/operators/executor.py

```diff
@@ -157,14 +151,13 @@ def execute_operator(operator_uri, ctx, params=None):
     )
 
 
-def _parse_ctx(ctx, params=None):
-    dataset = ctx.get("dataset", None)
-    view = ctx.get("view", None)
-    selected = ctx.get("selected", None)
-    selected_labels = ctx.get("selected_labels", None)
+def _parse_ctx(ctx=None, **kwargs):
+    if ctx is None:
+        ctx = {}
 
-    if params is None:
-        params = ctx.get("params", {})
+    ctx = {**ctx, **kwargs}  # don't modify input `ctx` in-place
```

Comment:
@ritch this is the other key change: it allows for `request_delegation` and `delegation_target` to be correctly passed through to the `ExecutionContext` using either of these syntaxes:

```py
foo.execute_operator(..., ctx={"request_delegation": True, ...})
foo.execute_operator(..., request_delegation=True)
```

Reply:
Great - and I'm assuming this is backwards compatible. Doesn't appear to be but I would be surprised if you missed that.

Reply:
yep, backwards compatible 👍